### DISCUSSION TITLE
alist 网页直接打开就访问了， 能加个密码吗？

### DISCUSSION THREAD

**Discussion by hehe036** [AlistGo/alist#1253] [General]
据说这个网页能解决 https://alist-doc.nn.ci/docs/meta/password 但是打不开了

**Comment by anwen-anyi** [comment]
？   https://alist-doc.nn.ci/docs/features/encrypt 文档不是有么？

**Comment by hehe036** [comment]
果然，原来是读题不细，谢谢
